db/seeds.rb

```diff
@@ -17,171 +17,307 @@
       plant.plant_stages.create!(
         name: stage_name,
-        duration: Faker::Number.between(from: 5, to: 30),
-        position: stage_names.find_index(stage_name) + 1
+        duration: rand(5..30),
+        position: stage_names.index(stage_name) + 1
       )
     end
   end
 
   # Requests
-  # db/seeds.rb
-
-  Request.create!(
-    requester_first_name: 'John',
-    requester_last_name: 'Doe',
-    requester_email: 'john.doe@example.com',
-    laboratory: 'My lab',
-    handler: User.first,
-    plant_stage: Plant.first.plant_stages.last,
-    name: 'My first request',
-    plant_name: Plant.first.name,
-    plant_stage_name: Plant.first.plant_stages.last.name,
-    comment: Faker::Movies::LordOfTheRings.quote,
-    due_date: Date.current + 3.months,
-    quantity: 50,
-    temperature: 'Chaud',
-    photoperiod: 8
-  )
-
-  Request.create!(
-    requester_first_name: 'Alice',
-    requester_last_name: 'Smith',
-    requester_email: 'alice.smith@example.com',
-    laboratory: 'My other lab',
-    name: 'My new request',
-    plant_name: Plant.last.name,
-    plant_stage_name: Plant.last.plant_stages.last.name,
-    plant_stage: Plant.last.plant_stages.last,
-    comment: Faker::Movies::LordOfTheRings.quote,
-    due_date: Date.current + 2.months,
-    quantity: 200,
-    temperature: 'Froid',
-    photoperiod: 16
-  )
-
-  # RequestDistributions
-  RequestDistribution.create!(
-    request: Request.first,
-    bench: Bench.first,
-    plant_stage: Request.first.plant_stage,
-    pot: Pot.first,
-    pot_quantity: 30,
-    positions_on_bench: [0, 0],
-    dimensions: [50, 50]
-  )
-
-  RequestDistribution.create!(
-    request: Request.first,
-    bench: Bench.first,
-    plant_stage: Request.first.plant_stage,
-    pot: Pot.second,
-    pot_quantity: 20,
-    positions_on_bench: [100, 50],
-    dimensions: [50, 20]
-  )
-
-  Request.first.update!(status: :accepted)
-
-  RequestDistribution.create!(
-    request: Request.second,
-    bench: Bench.second,
-    plant_stage: Request.second.plant_stage,
-    pot: Pot.second,
-    pot_quantity: 20,
-    positions_on_bench: [0, 10],
-    dimensions: [20, 30]
-  )
+  requests = []
+  5.times do |i|
+    plant = Plant.order('RANDOM()').first
+    plant_stage = plant.plant_stages.order('RANDOM()').first
+    requests << Request.create!(
+      requester_first_name: Faker::Name.first_name,
+      requester_last_name: Faker::Name.last_name,
+      requester_email: Faker::Internet.email,
+      laboratory: Faker::Science.scientist,
+      handler: User.first,
+      plant_stage: plant_stage,
+      name: "Requête #{i + 1}",
+      plant_name: plant.name,
+      plant_stage_name: plant_stage.name,
+      comment: Faker::Lorem.sentence,
+      due_date: Date.current + rand(1..6).months,
+      quantity: rand(10..100),
+      temperature: %w[Chaud Froid Extérieur].sample,
+      photoperiod: rand(6..18),
+      status: %i[pending refused completed].sample
+    )
+  end
```

Comment:
on peut utiliser `sample` plutôt que de faire un réordonnement aléatoire, et utiliser le map pour mettre directement le résultat dans l'objet `requests` sans avoir besoin d'utiliser le `<<` à chaque itération :) (c'est purement du pinaillage syntaxique hein, on est bien d'accord)

```suggestion
  requests = 5.times.map do |i|
    plant = Plant.all.sample
    plant_stage = plant.plant_stages.sample
    Request.create!(
      requester_first_name: Faker::Name.first_name,
      requester_last_name: Faker::Name.last_name,
      requester_email: Faker::Internet.email,
      laboratory: Faker::Science.scientist,
      handler: User.first,
      plant_stage:,
      name: "Requête #{i + 1}",
      plant_name: plant.name,
      plant_stage_name: plant_stage.name,
      comment: Faker::Lorem.sentence,
      due_date: Date.current + rand(1..6).months,
      quantity: rand(10..100),
      temperature: %w[Chaud Froid Extérieur].sample,
      photoperiod: rand(6..18),
      status: %i[pending refused completed].sample
    )
  end
```